$NetBSD: patch-ad,v 1.1 2007/12/01 19:25:23 rillig Exp $

Ft/Xml/XPath/_conversions.c:8: error: static declaration of 'PyNumber_NaN' follows non-static declaration
Ft/Lib/src/number.h:15: error: previous declaration of 'PyNumber_NaN' was here

--- Ft/Xml/XPath/_conversions.c.orig	2002-07-22 06:47:04.000000000 +0000
+++ Ft/Xml/XPath/_conversions.c	2007-12-01 19:22:44.000000000 +0000
@@ -5,7 +5,6 @@
 
 static PyObject *PyBoolean_Type;
 static PyCFunction PyBoolean_New;
-static PyObject *PyNumber_NaN;
 
 #define PyBoolean_Check(op) ((op)->ob_type == (PyTypeObject*)PyBoolean_Type)
 
